CryptoPrices/Sources/MyCoin/Sources/MyCoin/CoinStatisticsSection/CoinStatisticsSection.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  CoinStatisticsSection.swift
+//  MyCoin
+//
+//  Created by Khanh on 03/01/2023.
+//
+
+import Styleguide
+import SwiftUI
+
+struct CoinStatisticsSection: View {
+
+    var body: some View {
+        VStack {
+            coinStatisticsItemView(
+                title: Strings.MyCoin.MarketCap.title,
+                price: coinDetailItem.marketCap,
+                isPriceUp: coinDetailItem.isMarketCapChangePercentage24HUp,
+                percentage: coinDetailItem.marketCapChangePercentage24H
+            )
+            coinStatisticsItemView(
+                title: Strings.MyCoin.Ath.title,
+                price: coinDetailItem.ath,
+                isPriceUp: coinDetailItem.isAthChangePercentageUp,
+                percentage: coinDetailItem.athChangePercentage
+            )
+            coinStatisticsItemView(
+                title: Strings.MyCoin.Atl.title,
+                price: coinDetailItem.atl,
+                isPriceUp: coinDetailItem.isAtlChangePercentageUp,
+                percentage: coinDetailItem.atlChangePercentage
+            )
+        }
+        .padding(16.0)
+    }
+
+    private let coinDetailItem: CoinDetailItem
+
+    init(coinDetailItem: CoinDetailItem) {
+        self.coinDetailItem = coinDetailItem
+    }
+}
+
+private extension CoinStatisticsSection {
+
+    func coinStatisticsItemView(
+        title: String,
+        price: Decimal,
+        isPriceUp: Bool,
+        percentage: Double
+    ) -> some View {
+        HStack {
+            VStack(alignment: .leading) {
+                Text(title)
+                    .foregroundColor(Colors.textMedium.swiftUIColor)
+                    .font(Fonts.Inter.medium.textStyle(.caption))
+                    .padding(.vertical, 4.0)
+                
+                Text(price, format: .dollarCurrency)
+                    .foregroundColor(Colors.titleMedium.swiftUIColor)
+                    .font(Fonts.Inter.medium.textStyle(.callout))
+                    .padding(.vertical, 4.0)
```

Comment:
Just curious, why don't we define a fix spacing value directly into the `VStack` but we have to add the vertical padding individually like this? 🤔

Reply:
Thank you for the question 🙏 .
I had no specific purpose or tradeoff for adding the vertical padding individually per `Text` view instead of defining a fixed spacing value directly into the `VStack` , except for setting padding for it :D. IMO both approaches are fine for me.

Setting spacing value directly into `VStack` may reduce the repeated code 👍, a nice suggestion. Thank you for the raise 🙇 . I fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/84/commits/12ce15bee90af12af289e290ccc5eba464e95c1d.